container-startup

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -e
```

Comment:
There is in my opinion no need for this script. As you said, we can simply use `podman run python3 manage.py migrate` and so on.

Reply:
The main reason for this script is that we need to run a sequence of commands at startup. Please check `pcw/docker-entrypoint.sh` in the old `vault-docker` repository if you don't believe me.

Reply:
I also cannot comment if this is needed on every restart. It was done so before and I simply adapted the current workflow without major modifications, that's why it's there.